english/make_changes_GitHub/create_fork/pull_request/success.md

```diff
@@ -10,7 +10,13 @@ eyes to see that you are back in your room.
 You immediately run into your bathroom and had the greatest pee session ever.
 Everything was alright in the world.
 
+Thank goodness for GitHub! 
 
+But wait, you're an elite software engineer. The comforts of your home don't keep you entertained for long. You're  
+bored. You start thinking of the marshmallow adventure again. You long for that adrenaline rush and using your
+skills to get out of trouble. You look for the wormhole. It's still there.
 
+[Go back to marshmallow land](../../../marshmallow.md)
+
+[Create a new wormhole to go to Cooper Station](cooper-station/cooper-station.md)
 
-Thank goodness for GitHub! 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.